airbyte-webapp/src/components/StatusIcon/StatusIcon.tsx

```diff
@@ -66,7 +68,7 @@ const Value = styled.span`
   padding-left: 3px;
 `;
 
-const StatusIcon: React.FC<StatusIconProps> = ({ title, status, ...props }) => {
+const StatusIcon: React.FC<StatusIconProps> = ({ title, status = "error", ...props }) => {
```

Comment:
:information_source: Make `error` the default status, to not need to adjust all places this was used without a status.